userguide/virtualmachines.rst

```diff
@@ -460,18 +500,17 @@ for VNC.
    VNC Device
 
 
-The :guilabel:`Port` can be set to *0*, left empty for %brand% to
-assign a port when the VM is started, or set to a fixed, preferred
-port number.
+The :guilabel:`Port` can be set to *0* for %brand% to assign a port when
+the VM is started, or set to a fixed, preferred port number.
 
-Set :guilabel:`Wait to boot` to force the VNC client to wait
-until the VM has booted before attempting the connection.
+Set :guilabel:`Delay connection until VM is up` to force the VNC client
```

Comment:
This needs to be fixed in the UI and here.  It really means "Wait to boot the VM until a VNC connection is started", but we came up with better wording elsewhere.

Reply:
Updating to the proposed text from https://github.com/freenas/webui/pull/1807/files.